resources/CICD/fastlane/Constants/Constants.rb

```diff
@@ -0,0 +1,153 @@
+# frozen_string_literal: true
+
+class Constants
```

Comment:
We already have Fastlane in [our xctemplate](https://github.com/nimblehq/ios-templates/tree/develop/resources/xcode-templates/App.xctemplate/fastlane). Please remove all files related to fastlane

Reply:
There are some parts used in this PR that is missing in our current template. I will create another PR for fastlane files.